Update exchange-rate-canister.md (#1847)
dfinity/portal: docs/developer-docs/integrations/exchange-rate/exchange-rate-canister.md

> - If the request can be served from the cache, the actual cost is 200M cycles.
> - If both assets are fiat currencies, the cost is 200M cycles.

Reviewer: ```suggestion
- If the request can be served from the cache, the actual cost is 20M cycles.
- If both assets are fiat currencies, the cost is 20M cycles.
```

> - If one of the assets is a fiat currency or the cryptocurrency USDT, the cost is 0.26B cycles.
> - If both assets are cryptocurrencies, the cost is 0.5B cycles.

Reviewer: ```suggestion
- If one of the assets is a fiat currency or the cryptocurrency USDT, the cost is 260M cycles.
- If both assets are cryptocurrencies, the cost is 500M cycles.
```

> The remaining cycles are returned to the requesting canister. Note that at least 10M cycles are charged even in case of an error in order to mitigate the risk of a denial-of-service attack.

Reviewer: ```suggestion
The remaining cycles are returned to the requesting canister. Note that at least 1M cycles are charged even in case of an error in order to mitigate the risk of a denial-of-service attack.
```
